modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -212,41 +211,7 @@ kn workflow build --image dev.local/serverless-workflow-hello-world --verbose
 ----
 The `--verbose` flag is used to display the output of the build command. This flag is optional.
 
-For more information about Knative CLI, see xref:tooling/kn-plugin-workflow-overview.adoc[{context} plug-in for Knative CLI].
-
-After building the image, add `imagePullPolicy: IfNotPresent` to the `target/kubernetes/knative.yml` file. Your file should look similar to:
```

Comment:
Just a double-check. This was included in 1.24.x upstream but it wasn't merged on main, I wonder why.

See original PR for main
https://github.com/kiegroup/kogito-docs/pull/135
and the one for 1.24.x
https://github.com/kiegroup/kogito-docs/pull/142

Then again, cherry-pick for 1.25.x has it, but the one for 1.26.x does not. So which is the correct one? :)

Reply:
I am not sure if this was meant only for productization but It's cherry-picked in 1.26.x. @ricardozanini is this only required only in product?

Reply:
@MarianMacik this is specific for 1.24 because in that version `imagePullPolicy` was not being generated automatically.

Reply:
@ricardozanini I just checked and the code of the [original PR](https://github.com/kiegroup/kogito-docs/pull/135) is on 1.27.x. 

The original PR was cherry-picked to:
* 1.24.x with additional instructions that were specific to that version (this difference caused Marian's concern)
* 1.25.x identical to the original
* 1.26.x identical to the original

Reply:
Thanks @hbelmiro!